apps/drec-api/src/models/Organization.ts

```diff
@@ -11,7 +11,9 @@ export class IPublicOrganization {
   tradeRegistryCompanyNumber: string;
   vatNumber: string;
   status: OrganizationStatus;
+
   blockchainAccountAddress?: string;
+  blockchainAccountSignedMessage?: string;
```

Comment:
Could you shed some light what this property is? Because it doesn't match PR's description

Reply:
In order for a user to prove that he owns a specific blockchain address, it should sign a message with his private key.
This is the signed message that proves his ownership of the blockchain address.

I.e you can derive the blockchain address (public key) from the signed message.

We use this in Origin, but I see no UI implementation in this PR.

Reply:
UI will be added afterwards